src/solution_transfer/meshfree_interpolation.C

```diff
@@ -281,9 +281,21 @@ void InverseDistanceInterpolation<KDDim>::interpolate (const Point
 
   // Compute the interpolation weights & interpolated value
   const unsigned int n_fv = this->n_field_variables();
-  _vals.resize(n_fv); /**/ std::fill (_vals.begin(), _vals.end(), Number(0.));
-
-  Real tot_weight = 0.;
+  _vals.resize(n_fv);
+
+  std::fill (_vals.begin(), _vals.end(), Number(0.));
+  Real tot_weight(0.);
+  // The background value is optional
+  // If background value option is enabled, add it to the total weight
+  // If not, a "zero" weight is added
+  if (_background_eff_dist > 0.0)
+  {
+    const Number background_wt = _background_eff_dist * _background_eff_dist > std::numeric_limits<Real>::epsilon()
```

Comment:
We'd normally use some power of `TOLERANCE` here.

Reply:
I was trying to prevent doing 1/0. Maybe it's better to put a `libmesh_assert` here.

Reply:
That makes sense.  An assert wouldn't; it's not unreasonable to want to "interpolate" at zero distance.